libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -768,6 +770,9 @@ void RC_Channel::init_aux_function(const AUX_FUNC ch_option, const AuxSwitchPos
     case AUX_FUNC::CAMERA_AUTO_FOCUS:
     case AUX_FUNC::CAMERA_LENS:
 #endif
+#if AP_DDS_ENABLED
```

Comment:
```suggestion
#if AP_EXTERNAL_CONTROL_ENABLED
```

Reply:
why was this marked as resolved? looks to be still wrong

Reply:
It's fixed now.